### ISSUE TITLE
fr:us-phone: separate input static readonly doesn't have a label

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#2220]
Either:

- Use just 1 input, which we style dynamically.
- Add a label to static readonly input.

This per WCAG2 1.3.1 (A), reported by TotalValidator as E885.

**Comment by ebruchez** [comment]
#2219 now generates a `<span>` instead of a `<label>`, which doesn't fix the issue of course.

**Comment by ebruchez** [comment]
After #2216, we have a label which points to the HTML input with `xxbl:label-for="html-input"`.

But in static readonly, `html-input` is hidden, and another `<xf:input>` is used to be shown as static readonly. That one doesn't have a label.

In passing, it is annoying that we have to mix code between regular and static readonly, and discriminate in XForms. If we had separate templates or bindings, we could do better.

**Comment by ebruchez** [comment]
This is a bigger issue right now: in static readonly, we don't output an HTML `<input>` element, but instead we input a `<span>`. TotalValidator would complain even if we have a `<label>` pointing to the `<span>`. 

If we keep a `<span>`, we should:

- use `aria-labelledby` on the `<span>`
- use `<span>` instead of `<label>`
- add an id on that new `<span>`

We could also switch to using an `<output>`:

- switch
- make the `<label>` point to the output
